### ISSUE TITLE
[theme-gen] Provide a way to use other colors in the color palette as contrast variables

### ISSUE THREAD

**Issue by nasheomirro** [nasheomirro/stratum#49]
Because of issue #47 as well as the fact that some shades struggle to meet the required contrast value for WCAG. We should allow a way to use other values in the theme for contrast (most commonly the `surface` color set)

**Comment by nasheomirro** [comment]
Also, maybe visually tell a user that a color needs more contrast, or just show the contrast value as good or bad.

**Comment by nasheomirro** [comment]
or maybe we don't have to modify the `stratum-theme` package. We should be able to provide a way to change contrast colors without adding it to the theme object.
